src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -1398,7 +1398,7 @@ public void Process_StartTest()
             string domain = "thisDomain";
             SecureString password = AsSecureString("Value");
 
-            Process p = Process.Start(currentProcessName, userName, password, domain);
+            Process p = Process.Start(currentProcessName, userName, password, domain); // This writes junk to the Console but with this overload, we can't prevent that.
```

Comment:
The "junk" is from invoking dotnet.exe without arguments?  Can we invoke something else that doesn't have such spew? (not for this PR to release/2.0, but for a subsequent change in master)